.github/workflows/pull_request_verification.yml

```diff
@@ -5,10 +5,59 @@ on:
     branches: '**'
 
 jobs:
-  build:
 
+  branch_name:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: gradle
+
+      - name: Check correct branch name
+        run: |
+          tag=${GITHUB_HEAD_REF#refs/*/v}
+          if [[ $tag =~ [a-z]{2,}\/[0-9]+-[A-Za-z-]+$ ]]
+          then
+            echo "::notice::Tag is valid — $tag"
+            echo "{tag}={GITHUB_HEAD_REF#refs/*/v}" >> $GITHUB_OUTPUT
+          else
+            echo "::error::Tag is invalid — $tag"
+            echo "::error::Tag should looks like - \$acronym/\$issue_number-\$description"
+            exit 1
+          fi    
```

Comment:
is it possible to easily extract this check expression to some function/script?

Reply:
I will check it

Reply:
I created a file for this script.